composer.json

```diff
@@ -22,7 +22,7 @@
         }
     },
     "require": {
-        "php": ">=5.6.5"
+        "php": ">=5.6.5 || >=7.0"
```

Comment:
yes it should

the strange thing is when i run `composer remove werkspot/enum dev-master`
this will result in
``` 
Problem 1
    - This package requires php >=5.6.5 but your PHP version (7.0.3) does not satisfy that requirement.
```

aldough `composer remove werkspot/enum dev-LauLaman-patch-1` ( with `|| >=7.0`) works just fine